### ISSUE TITLE
KCG not working on V8.4 study

### ISSUE THREAD

**Issue by PPlessiez** [AntaresSimulatorTeam/Antares_Simulator#1104]
**Description**
When executing the Kirchhoff's constraints generator on the attached study in V8.4, an error message is displayed:

![2023-01-19 15_32_06-Test_Capa - Antares Simulator](https://user-images.githubusercontent.com/53614116/213469868-bbaea914-b751-4d18-94a8-15368bf88f14.png)

As a result, the corresponding binding constraints are created, but they are empty (no weight attached to any link).

From the error message, there seems to be an issue in the path of the "parameters" files which are retrieved: for example, the parameters file for BE-Conso link is located in
_D:\Users\plessiezpau\Documents\10 - Réseau\Test_Capa\input\links\be\conso_parameters.txt_, not in 
_D:\Users\plessiezpau\Documents\10 - Réseau\Test_Capa\input\links\be\conso_parameters.txt\conso_parameters.txt_

**Screenshots and additional files**
[Test_Capa.zip](https://github.com/AntaresSimulatorTeam/Antares_Simulator/files/10457677/Test_Capa.zip)

**OS and version**
 - OS: Windows
 - Antares Simulator version: 8.4

**Comment by payetvin** [comment]
The errors are related to JIT, thus only causing problem in the GUI.
I didn't see any differences in the binding constraints generated from the GUI and from the separated binary, which is the expected behavior. 

Are you sure the binding constraints generated are wrong ? If so the problem is deeper and we'll need time to investigate it.

**Comment by payetvin** [comment]
Backtrace of the I/O error for future debugging
```[0] from 0x00005555558a871c in Antares::Matrix<double, double>::internalLoadCSVFile(Yuni::CString<0u, true> const&, unsigned int, unsigned int, unsigned int, Yuni::CString<4096u, true>*)+402 at /home/payetvin/Antares_Simulator/src/libs/antares/array/matrix.hxx:1030
[1] from 0x00005555558a7cc1 in Antares::Matrix<double, double>::loadFromCSVFile(Yuni::CString<0u, true> const&, unsigned int, unsigned int, unsigned int, Yuni::CString<4096u, true>*)+171 at /home/payetvin/Antares_Simulator/src/libs/antares/array/matrix.hxx:403
[2] from 0x00005555559687ba in Antares::Matrix<double, double>::loadAllJITData() const+198 at /home/payetvin/Antares_Simulator/src/libs/antares/study/constraint/../area/../../array/matrix.hxx:1299
[3] from 0x000055555596868d in Antares::Matrix<double, double>::forceReload(bool) const+55 at /home/payetvin/Antares_Simulator/src/libs/antares/study/constraint/../area/../../array/matrix.hxx:342
[4] from 0x0000555555add261 in Antares::Data::AreaLink::forceReload(bool) const+43 at /home/payetvin/Antares_Simulator/src/libs/antares/study/area/links.cpp:909
[5] from 0x0000555555a42900 in Antares::CBuilder::update(bool)+666 at /home/payetvin/Antares_Simulator/src/solver/constraints-builder/cbuilder.cpp:122
[6] from 0x0000555555a43a56 in Antares::CBuilder::runConstraintsBuilder(bool)+54 at /home/payetvin/Antares_Simulator/src/solver/constraints-builder/cbuilder.cpp:278
```

**Comment by flomnes** [comment]
Hello @PPlessiez, v8.5.0 should fix this, can you confirm ?

**Comment by PPlessiez** [comment]
I confirm that the problem was solved following the additional developments from @payetvin, so if this is included in V8.5.0, then this version should fix the issue!
